test/functional/wallet_importmulti.py

```diff
@@ -467,6 +467,67 @@ def run_test (self):
         import_pub2 = self.nodes[0].getaddressinfo(addr2)['pubkey']
         assert_equal(pub2, import_pub2)
 
+        # Import some public keys to the keypool of a no privkey wallet
```

Comment:
Commit "Fetch keys from keypool when private keys are disabled"

Could move test to a different commit?

Reply:
Done